build.cake

```diff
@@ -207,6 +213,7 @@ Task("Publish-NuGet")
     .WithCriteria(() => parameters.IsMainCakeRepo)
     .WithCriteria(() => parameters.IsMainCakeBranch)
     .WithCriteria(() => parameters.IsTagged)
+    .ContinueOnError()
```

Comment:
@patriksvensson yeah, I wasn't sure what would happen there.  Will need to test it out to see what happens.  If it does, any suggestions on how best to handle this?